servlet/src/main/java/io/undertow/servlet/spec/AsyncContextImpl.java

```diff
@@ -145,6 +145,14 @@ public boolean hasOriginalRequestAndResponse() {
                 servletResponse instanceof HttpServletResponseImpl;
     }
 
+    public boolean isInitialRequestDone() {
+        return initialRequestDone;
+    }
+
+    public void setInitialRequestDone(boolean initialRequestDone) {
```

Comment:
@stuartwdouglas  Nothing calls this.

Reply:
oops, I generated the setter by accident.